src/cc/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set_target_properties(bcc-static PROPERTIES OUTPUT_NAME bcc)
 # BPF is still experimental otherwise it should be available
 #llvm_map_components_to_libnames(llvm_libs bpf mcjit irreader passes)
 llvm_map_components_to_libnames(llvm_libs bitwriter bpfcodegen irreader linker
-  mcjit objcarcopts option passes x86codegen)
+  mcjit objcarcopts option passes x86codegen powerpccodegen)
```

Comment:
No I use it in iomodules as well, since right now this is the only way to get keys in/out of the maps without having the struct layout known in the particular binding. I don't like it, this has been itching me for a while, and as soon as I find a way to do this more cleanly I will remove the old way.